{"name": "Проверка сетчика ингредиентов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:197:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:527:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_base.TestBaseFunctionality object at 0x7f6b9f546910>\ndriver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"16247201-e3d0-43c1-8944-4bbb3aab9be1\")>\n\n    @title(\"Проверка сетчика ингредиентов\")\n    def test_ingredients_counter(self, driver):\n        page = MainPage(driver)\n>       page.add_buns_to_order()\n\ntests/test_base.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:42: in add_buns_to_order\n    self.wait_for_load_element(MainPageLocators.BUN_IN_ORDER)\npages/base_page.py:15: in wait_for_load_element\n    WebDriverWait(self.driver,TIMEOUT).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"16247201-e3d0-43c1-8944-4bbb3aab9be1\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f6b9f412a60>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:197:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:527:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n.venv/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "steps": [{"name": "Добавление булочки в заказ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:197:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:527:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/home/marina/PycharmProjects/Diplom_3/.venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/marina/PycharmProjects/Diplom_3/pages/main_page.py\", line 42, in add_buns_to_order\n    self.wait_for_load_element(MainPageLocators.BUN_IN_ORDER)\n  File \"/home/marina/PycharmProjects/Diplom_3/pages/base_page.py\", line 15, in wait_for_load_element\n    WebDriverWait(self.driver,TIMEOUT).until(\n  File \"/home/marina/PycharmProjects/Diplom_3/.venv/lib/python3.8/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1746969570447, "stop": 1746969581254}], "attachments": [{"name": "log", "source": "6a06b9bb-8959-41a4-a922-b3e6914b5482-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1746969568198, "stop": 1746969581255, "uuid": "359b7854-6804-4ca8-829f-fe58d8dcbd66", "historyId": "34cfa39ee1898934c3823916f50edb0f", "testCaseId": "2cbdb228fd8006bf2369c9e1d4e43b0b", "fullName": "tests.test_base.TestBaseFunctionality#test_ingredients_counter", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_base"}, {"name": "subSuite", "value": "TestBaseFunctionality"}, {"name": "host", "value": "ws"}, {"name": "thread", "value": "1038636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_base"}]}